src/core/cgroup.c

```
@@ -293,8 +293,13 @@ static int whitelist_device(const char *path, const char *node, const char *acc)
         assert(acc);
 
         if (stat(node, &st) < 0) {
-                log_warning("Couldn't stat device %s", node);
-                return -errno;
+                /* path starting with "-" must be silently ignored */
+                if (startswith(node, "-"))
+                        return 0;
```

i think we should ignore the error only if ENOENT.

---

Good catch. I updated, and force-pushed. Thanks a lot!